src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingTownHall.java

```diff
@@ -60,6 +60,17 @@ public int getMaxBuildingLevel()
         return MAX_BUILDING_LEVEL;
     }
 
+    /**
+     * Townhall does not house any citizens.
+     *
+     * @return 0
+     */
+    @Override
+    public int getMaxInhabitants()
+    {
+        return 0;
+    }
+
```

Comment:
we should just be able to remove the building home reference.

Reply:
broke sth UI related when I tried replacing it with abstractbuilding